web/packages/extension/src/content.ts

```diff
@@ -174,6 +174,7 @@ function isXMLDocument(): boolean {
             autoplay: options.autostart ? "on" : "auto",
             unmuteOverlay: options.autostart ? "hidden" : "visible",
             splashScreen: !options.autostart,
+            maxExecutionDuration: options.maxExecutionDuration,
```

Comment:
This line is unnecessary due to the `...options` above.

Reply:
I completely forgot about that, thank you.